plugins/auth/fps_auth/db.py

```diff
@@ -18,7 +18,7 @@
 
 auth_config = get_config(AuthConfig)
 
-jupyter_dir = Path.home() / ".local" / "share" / "jupyter"
+jupyter_dir = Path.home() / ".local/share/jupyter"
```

Comment:
From what I can read, I think it works because Python does some tricks to convert forward slashes to backslashes if needed when opening files, but the best practice is still to use `pathlib`'s `/` operator.